app/src/coin.h

```diff
@@ -42,8 +42,9 @@ typedef enum {
 
 // Coin Specific
 #define PK_ADDRESS_TYPE                     COIN_ADDR_TYPE_KUSAMA
-#define SUPPORTED_TX_VERSION                LEDGER_MAJOR_VERSION
-#define SUPPORTED_SPEC_VERSION              (2000+LEDGER_MINOR_VERSION)
+#define SUPPORTED_TX_VERSION_CURRENT        LEDGER_MAJOR_VERSION
+#define SUPPORTED_TX_VERSION_PREVIOUS       (LEDGER_MAJOR_VERSION - 1)
+#define SUPPORTED_SPEC_VERSION              LEDGER_MINOR_VERSION
```

Comment:
We may have a problem here.

Unfortunately, ledger does not support version numbers > 255. But Kusama uses Major version numbers >2000, for this reason the value in the makefile needs to be increased by 2000. Removing that will probably result in incorrect checks.

Reply:
ok, fixed it